api/src/guards/throttler.guard.ts

```diff
@@ -0,0 +1,19 @@
+import { ThrottlerGuard } from '@nestjs/throttler';
+import { ExecutionContext, Injectable } from '@nestjs/common';
+import { ThrottlerLimitDetail } from '@nestjs/throttler/dist/throttler.guard.interface';
+
+@Injectable()
+export class ThrottleGuard extends ThrottlerGuard {
+  protected async getTracker(req: Record<string, any>): Promise<string> {
+    console.log('7:', req.ips.length ? req.ips : req.ip);
```

Comment:
What does the "7" signify here?

Reply:
ah that was just the line number, that log will go away after I test a bit on core's hosted instance

so I think the 7 can stick around for a lil bit